lib/modules/genre.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../item'
+
+class Genre < Item
+  attr_reader :id, :items
+
+  def initialize(name, items)
+    @id = Random.rand(1..100)
+    @name = name
+    @items = items
+    super(false, Date.today.to_s)
+  end
```

Comment:
- [ ] Excellent job building your constructor for the Genre class 🤓. However, since Genre does not inherit from any class, the inherited attributes `super(false, Date.today.to_s)` are not needed.

- [ ] Also, please kindly make sure that your `items` attribute is type array like requested in the UML map below 👇🏽.

![cr2](https://user-images.githubusercontent.com/116701485/234331045-92d99305-0a53-400d-b1e8-03bdf300b428.png)

A good idea of how the constructor may look after the changes is 👇🏽.
```
def initialize(name)
    @id = Random.rand(1..100)
    @name = name
    @items = []
  end
```